lib/tty/prompt/choices.rb

```diff
@@ -113,6 +106,16 @@ def pluck(name)
       def find_by(attr, value)
         find { |choice| choice.public_send(attr) == value }
       end
+
+      protected
+
+      # The actual collection choices
+      #
+      # @return [Array[Choice]]
+      #
+      # @api private
+
+      attr_reader :choices
```

Comment:
@piotrmurach As a stray reader of this code I can witness it is confusing to see "different" choices, one stumbles upon "which choices are meant here and there?". This is obviously not urgent or something, but we might consider renaming it eventually if there is a good name for it (`raw_choices` sounds a bit dull-ish).

Reply:
That's a good point. It's not ideal but equally, I lack inspiration for a better name.